app/main/Constants/Referents.php

```diff
@@ -32,6 +32,8 @@ class Referents
 
     public const uk = ['MrAlKuz'];
 
+    public const vi = ['dinhquochan'];
```

Comment:
@caouecs, we have a new person to join our team 😊

Reply:
done